capellambse/model/modeltypes.py

```diff
@@ -200,3 +200,11 @@ class AggregationKind(_StringyEnumMixin, _enum.Enum):
     ASSOCIATION = _enum.auto()
     AGGREGATION = _enum.auto()
     COMPOSITION = _enum.auto()
+
+
+class ScenarioKind(_StringyEnumMixin, _enum.Enum):
+    UNSET = _enum.auto()
```

Comment:
Here is also no _useless_ docstring but other enums except `AggregationKind` also have none.